test/system/702-artifact.bats

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bats   -*- bats -*-
```

Comment:
Why 702 not 700?

Reply:
test/system/700-play.bats is still around